easybuild/easyconfigs/a/ASE/ASE-3.24.0-iimkl-2023a.eb

```
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.24.0'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language.
+
+From version 3.20.1 we also include the ase-ext package, it contains optional reimplementations
+in C of functions in ASE.  ASE uses it automatically when installed."""
+
+toolchain = {'name': 'iimkl', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('SciPy-bundle', '2023.07'),
+    ('Flask', '2.3.3'),
+    ('matplotlib', '3.7.2'),
+    ('Tkinter', '%(pyver)s'),     # Needed by GUI of ASE
+    ('spglib-python', '2.1.0'),  # optional
```

```suggestion
    ('matplotlib', '3.7.2'),
    ('Flask', '2.3.3'),           # Needed by web interface of ase.db module
    ('Tkinter', '%(pyver)s'),     # Needed by GUI of ASE
    ('spglib-python', '2.1.0'),   # optional
```

From their docs, `Flask` is optional and specifically for the Web ase.db interface

---

Does this really matter?  

If I change this, then it should be changed in foss/2023a and foss/2024a also, the similar PRs were merged into these chains yesterday.  And I don't really feel like bothering the busy developers with another PR which just changes a comment.

---

... but I otherwise agree with you, and will change it when we release ASE 3.25.0 :-)

---

It's a minor thing, it's just good to keep track of which modules are strictly required for e.g. future refactors or incompatibilities. You could add this change to all affected files to this PR.

I am not a maintainer, so I won't decide if it's important enough or not :)